Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -1212,6 +1212,7 @@ - (void)viewDidAppear:(BOOL)animated {
     }
     [self updateNavigationBarSubtitleLabel];
     [MarkIdentityAsSeenJob runWithThread:self.thread];
+    [ProfileFetcherJob runWithThread:self.thread networkManager:self.networkManager];
```

Comment:
Brainstorming - would we also want do this...

* ...When we add a member to a group?
* ...When we "export with Signal" to a thread? Or I guess that's redundant, since we'll learn of the identity change when we do the send?